### DISCUSSION TITLE
Temporary solution for Simulator side-service and settings-app errors

### DISCUSSION THREAD

**Discussion by MyoungXUE** [zepp-health/zeppos-forum#272] [Announcements]
Due to CDN distribution issues in some regions, the Side-service and settings-app of the emulator cannot start normally, resulting in errors like the following figure.

![image](https://github.com/zepp-health/zeppos-forum/assets/107526244/7c3fd2d4-adba-445a-a001-00481fe0e7d5)

In response to this issue, we are communicating with Cloud as a Service Provider to solve the problem as soon as possible. If you need to use it normally before this, please refer to the following step.

1. Remove Simulator Client
2. Install the latest version 1.2.3
  - Download link https://docs.zepp.com/docs/guides/tools/simulator/download/

**Comment by melianmiko** [comment]
Maybe a better solution will be packing these JavaScript libraries with the simulator, for fully offline use?

**Reply by MyoungXUE** [reply]
This is a good idea, and we will optimize it based on your suggestions.
